mne/viz/evoked.py

```diff
@@ -1824,9 +1824,10 @@ def plot_evoked_joint(
         axes are passed make sure to set ``title=None``, otherwise some of your
         axes may be removed during placement of the title axis.
     %(picks_all)s
-    exclude : None | list of str | 'bads'
+    exclude : list of str | 'bads'
```

Comment:
should we change this to something like `exclude : empty list | list of str | 'bads'`?

Reply:
I don't think so `empty list` is covered by `list of str` well enough I think